compiler/rustc_typeck/src/expr_use_visitor.rs

```diff
@@ -28,19 +26,18 @@ use crate::mem_categorization as mc;
 /// This trait defines the callbacks you can expect to receive when
 /// employing the ExprUseVisitor.
 pub trait Delegate<'tcx> {
-    // The value found at `place` is either copied or moved, depending
+    // The value found at `place` is moved, depending
     // on `mode`. Where `diag_expr_id` is the id used for diagnostics for `place`.
     //
+    // Use of a `Copy` type in a ByValue context is considered a use
+    // by `ImmBorrow` and `borrow` is called instead.
```

Comment:
```suggestion
    // Use of a `Copy` type in a ByValue context is considered a use
    // by `ImmBorrow` and `borrow` is called instead. This is because
    // a shared borrow is the "minimum access" that would be needed
    // to perform a copy.
```